lib/modules/apostrophe-login/index.js

```diff
@@ -168,10 +180,11 @@ module.exports = {
           { email: username }
         ],
         disabled: { $ne: true }
-      });
+      }).toObject();
+
       if (!user) {
         await Promise.delay(1000);
-        throw 'invalid';
+        return false;
```

Comment:
Fine with this, it is passporty. But the doc comment should be fixed to cover this case.

Reply:
👍 I added updated the comment.